ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -1131,6 +1157,17 @@ for target in "${TARGET_RELEASES[@]}"; do
 	health_check
     fi
 
+    run_command "oc get -o json nodes.config.openshift.io cluster | jq -r .spec.cgroupMode"
+    # From OCP 4.19, we do not support cgroupmode v1
+    # So update the 'cgroupMode' in the 'cluster' object of nodes.config.openshift.io resource type to 'v2'
+    if [[ "${TARGET_MINOR_VERSION}" -eq "18" ]] && [[ "$(oc get -o json nodes.config.openshift.io cluster | jq -r .spec.cgroupMode)" == "v1" ]]; then
```

Comment:
`TARGET_MINOR_VERSION` is 19?

Reply:
no, it indeed 4.18

Reply:
4.19 not support cgroupMode v1, so we need to upgrade it after the cluster upgraded to 4.18.

Reply:
Yes, my understanding is if your current cluster is 4.18 and your target cluster is 4.19 then this step is supposed to run.
I assume when the variable `TARGET_MINOR_VERSION` is 4.19, then you are currently on 4.18 cluster and this step is invoked. When `TARGET_MINOR_VERSION` is 4.18, you are actually a 4.17 cluster.

https://github.com/openshift/release/blob/3f424b4e2a0b1b2c81eaf48a2b25903cd55aa25e/ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh#L1111

Reply:
Because this check is after `upgrade`, so we can use "${TARGET_MINOR_VERSION}" -eq "18"